ios/engine/KMEI/KeymanEngine/Classes/Model/InstallableLexicalModel.swift

```diff
@@ -14,11 +14,24 @@ struct InstallableConstants {
 
 /// Mainly differs from the API `LexicalModel` by having an associated language.
 public struct InstallableLexicalModel: Codable, LanguageResource {
+  // Details what properties are coded and decoded re: serialization.
+  enum CodingKeys: String, CodingKey {
+    case id
+    case name
+    case lgCode = "languageID"  // Redirects the old plain-property to something we can wrap with accessors.
+    case version
+    case isCustom
+  }
+
   public private(set) var id: String
   public var name: String
-  public private(set) var languageID: String
+  private var lgCode: String
   public var version: String
   public var isCustom: Bool
+
+  public var languageID: String {
+    return lgCode.lowercased()
```

Comment:
Does this handle kmp.json info from lexical model .kmp packages? (If it used `str-Latn`)

Reply:
The JSON's `languageID` property will be rerouted to the Swift `lgCode` property.  This Swift `languageID` is (now) a wrapper around that which forces lowercasing, allowing it to always match the `languageID` from the keyboard.

By handling it this way, it will still work for ad-hoc and previously-installed language resources.  (Resources already on a user's device may not be lowercased, but this wrapper will still force lowercasing throughout the rest of the app.)